app/js/DockerRunner.js

```diff
@@ -91,6 +91,13 @@ module.exports = DockerRunner = {
       image = `${Settings.texliveImageNameOveride}/${img[2]}`
     }
 
+    if (
+      Settings.clsi.docker.allowedImages &&
+      !Settings.clsi.docker.allowedImages.includes(image)
+    ) {
+      return callback(new Error('image not allowed'))
+    }
```

Comment:
We could move this check above the override and provide the quay names in the env var.

Reply:
Yes, that would be fine.

Reply:
Done in https://github.com/overleaf/clsi/pull/183/commits/0cecf2656981141e8951ac8aafca6a1c07c972a9. google-ops change: https://github.com/overleaf/google-ops/compare/d763fc0a1cd1832500f00fe84200ba55e51cb40b..87e5057b926400929f2efc97509529d1e221a444